Content.Client/_RMC14/Vendors/CMAutomatedVendorBui.cs

```diff
@@ -40,9 +47,25 @@ protected override void Open()
             for (var sectionIndex = 0; sectionIndex < vendor.Sections.Count; sectionIndex++)
             {
                 var section = vendor.Sections[sectionIndex];
+
+                var validJob = true;
+                if (_player.LocalSession != null && _mind.TryGetMind(_player.LocalSession.UserId, out var mindId))
+                {
+                    foreach (var job in section.Jobs)
+                    {
+                        if (!_job.MindHasJobWithId(mindId, job.Id))
```

Comment:
This needs to be checked on the server too, not just the client, otherwise a modified client could vend anything